ci-integration/run-test-docker.sh

```diff
@@ -70,7 +70,9 @@ run_test(){
     base_filename="$(basename "$filename")"
     # Start the docker run module.
     docker run -e "IGNORE_ENV_CHECK=1" --name "$base_filename" \
-        -t volttron_test_image pytest "$filename" > "$base_filename.result.txt" 2>&1 &
+            -t --network="host" -v /var/run/docker.sock:/var/run/docker.sock volttron_test_image \
```

Comment:
Based on feedback from @craig8 from today's meeting, use same docker run command for all test runs. The additional options won't negatively affect the other tests.